client/events/inputs.mjs

```diff
@@ -37,7 +37,7 @@ export default (game) => {
       game.profile.enableSelectionButton();
       game.profile.setSelectedName(obj.displayName);
 
-      game.socket.emit("selectPlayer", {
+      game.socket.emit("player:select", {
```

Comment:
Because of `player:selection:drop` below, change `player:select` to `player:selection:add`